.gitignore

```diff
@@ -1,17 +1,5 @@
-#VSCode
-/.vscode
-
 #C++
 /build
-/builddir
-*.so
-
-#Python
-__pycache__/
-*.pyc
```

Comment:
What is he reason behind not ignoring these python bytecodes? Is it for faster startup time?

Reply:
Oh I meant to move that to the genai directory which has its own gitignore file.